test/dotnet-new3.UnitTests/DotnetNewHelp.cs

```diff
@@ -0,0 +1,182 @@
+  --package <PACKAGE>            Filters the templates based on NuGet package ID. Applies to --search.
+  --columns <COLUMNS_LIST>       Comma separated list of columns to display in --list and --search output.
+                                 The supported columns are: language, tags, author, type.
+  --columns-all                  Display all columns in --list and --search output.
+  --tag <TAG>                    Filters the templates based on the tag. Applies to --search and --list.";
+        private const string ConsoleHelp =
+@"Console Application (C#)
+Author: Microsoft
+Description: A project for creating a command-line application that can run on .NET Core on Windows, Linux and macOS
+Options:                                                                             
+  -f|--framework  The target framework for the project.                              
+                      net6.0           - Target net6.0                               
+                      net5.0           - Target net5.0                               
+                      netcoreapp3.1    - Target netcoreapp3.1                        
+                      netcoreapp3.0    - Target netcoreapp3.0                        
+                      netcoreapp2.2    - Target netcoreapp2.2                        
+                      netcoreapp2.1    - Target netcoreapp2.1                        
+                      netcoreapp2.0    - Target netcoreapp2.0                        
+                      netcoreapp1.0    - Target netcoreapp1.0                        
+                      netcoreapp1.1    - Target netcoreapp1.1                        
+                  Default: net6.0                                                    
+
+  --langVersion   Sets the LangVersion property in the created project file          
+                  text - Optional                                                    
+
+  --no-restore    If specified, skips the automatic restore of the project on create.
+                  bool - Optional                                                    
+                  Default: false                                                     ";
+
+        private const string ClassLibHelp =
+@"Class Library (C#)
+Author: Microsoft
+Description: A project for creating a class library that targets .NET Standard or .NET Core
+Options:                                                                             
+  -f|--framework  The target framework for the project.                              
+                      net6.0            - Target net6.0                              
+                      netstandard2.1    - Target netstandard2.1                      
+                      netstandard2.0    - Target netstandard2.0                      
+                      net5.0            - Target net5.0                              
+                      netcoreapp3.1     - Target netcoreapp3.1                       
```

Comment:
These test will break whenever we make a change to the templates or whenever we change the formatting of the console output. This will just be extra work for us.

 Would it be better to check only for the existence of:
- template name
- one of the options
- description of one of the option.

This way, we won't be affected by the formatting changes, or when we add a new parameter or choice.

Reply:
I would like to catch formatting changes here as well. 
if they are intentional, changing the string won't take much time.